main.go

```diff
@@ -146,11 +162,12 @@ func resetPasswords(f *excelize.File, input *Input, portal *Portal) (err error)
 	return nil
 }
 
-func rotate(input *Input, portal *Portal) error {
-	f, err := excelize.OpenFile(input.Filename)
+func rotate(input *Input, portal *Portal, client S3ClientAPI) error {
+	f, err := downloadFile(input, client)
 	if err != nil {
 		return err
 	}
+	defer os.RemoveAll(path.Dir(f.Path))
 
 	// true means "block action"
 	err = f.ProtectSheet(input.AutomatedSheetName, &excelize.FormatSheetProtection{
```

Comment:
Is there any purpose to doing this anymore?

Reply:
I think so. The sheet will downloaded from the bucket and emailed to testers. The testers must not be able to change the automated sheet.